packages/flutter_tools/lib/src/devfs.dart

```diff
@@ -312,11 +308,11 @@ class _DevFSHttpWriter implements DevFSWriter {
   static const int kMaxInFlight = 3;
 
   int _inFlight = 0;
-  Map<Uri, DevFSContent> _outstanding = <Uri, DevFSContent>{};
-  Completer<void> _completer = Completer<void>();
+  late Map<Uri, DevFSContent> _outstanding;
+  late Completer<void> _completer;
```

Comment:
Because the initial values were never used--`_completer` would be overwritten on (old line numbers) line 322, `_outstanding` on 323. If we're accessing either of these before that initialization it's probably a bug we should know about.